tests/run_tests.py

```diff
@@ -39,7 +46,7 @@
 epilog = '''
 -------------------------------------------------------------------------------
 The VIC Test Suite was developed by Joe Hamman at the University of Washington
```

Comment:
Since others will be contributing change it to be more inclusive or change `developed` to `initiated`.